docs/rst-styleguide.rst

```diff
@@ -0,0 +1,434 @@
+============================
+ReStructuredText Style Guide
```

Comment:
Capitalize only the first word of headings, except here it is "reStructuredText"